src/main/webapp/app/shared/chart/chart-category-filter.ts

```diff
@@ -102,9 +102,7 @@ export class ChartCategoryFilter extends ChartFilter {
             if (!exercise.categories) {
                 return this.includeExercisesWithNoCategory;
             }
-            return exercise
-                .categories!.flatMap((category: ExerciseCategory) => this.filterMap.get(category.category!)!)
-                .reduce((value1: boolean, value2: boolean) => value1 || value2);
+            return exercise.categories!.flatMap((category: ExerciseCategory) => this.filterMap.get(category.name!)!).reduce((value1: boolean, value2: boolean) => value1 || value2);
```

Comment:
No change in logic, this line is now only wrapped because it is a few chars shorter after renaming